tests/test_bash_unicode_output.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
As far as I understand, you added tests/test_bash_unicode_output.sh just to run (manually) it with bash_unit and check if you can see the bullets.

If that is correct, I suggest you just remove this file and relies on the ones already existing in the getting_started directory.

Regards,